ui/raidboss/data/06-ew/raid/p1s.ts

```diff
@@ -45,6 +45,12 @@ const fireLightOutputStrings = {
     cn: '站在光',
     ko: '흰색 바닥 위에 서기',
   },
+  oppositePlayer: {
+    en: 'Stand on color opposite of ${player}',
```

Comment:
These are both pretty wordy compared to the original.  What about just "Opposite of player/party"? Or, am I overthinking this?

Reply:
How about Opposite color of player/party? I feel that without 'color' in it people may conflict with cardinals...

Reply:
opposite ${player} color / opposite party color?

Reply:
Oh you already adjusted it to that.  That's fine.  o7